css/styles.css

```diff
@@ -581,9 +579,191 @@ nav {
   .recent-works-line {
     display: none;
   }
+
+  /* Project details Modal */
+
+  .project-modal {
+    display: none;
+    position: fixed;
+    z-index: 1;
+    padding-top: 100px;
+    left: 0;
+    top: 0;
+    width: 100%;
+    height: 100%;
+    overflow: auto;
+    background-color: rgb(0, 0, 0);
+    background-color: rgba(0, 0, 0, 0.4);
+  }
+
+  .modal-header {
+    margin-bottom: -10px;
+  }
+
+  .modal-content {
+    background-color: #fff;
+    margin: auto;
+    padding: 20px;
+    border: 1px solid #888;
+    width: 90%;
+  }
+
+  .project-img {
+    width: 100%;
+    margin-right: 25px;
+    margin-bottom: 25px;
+  }
+
+  .project-buttons li {
+    display: flex;
+    flex-direction: row-reverse;
+    margin-right: 20px;
+    padding: 0 20px;
+  }
+
+  .project-buttons li img {
+    height: 20px;
+    position: relative;
+    top: 25%;
+    bottom: 25%;
+    left: 5%;
+  }
+
+  .modal-body {
+    display: flex;
+    flex-direction: column-reverse;
+    margin-top: 40px;
+  }
+
+  .modal-heading {
+    color: #172b4d;
+    font-family: 'Crete Round', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 32px;
+    line-height: 44px;
+    padding-bottom: 80px;
+  }
+
+  .modal-details {
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 30px;
+    color: #344563;
+    margin-bottom: 20px;
+  }
+
+  .close-button {
+    font-family: 'Inter', sans-serif;
+    position: relative;
+    right: 0;
+    top: -150px;
+    color: #67798e;
+    float: right;
+    font-size: 16px;
+    cursor: pointer;
+  }
+
+  .close-button:focus {
+    color: #272f38;
+    text-decoration: none;
+    cursor: pointer;
+  }
+
+  /* End Project details Modal */
 }
 
 @media screen and (min-width: 768px) {
+  /* Project details Modal */
+
+  .project-modal {
+    display: none;
+    position: fixed;
+    z-index: 1;
+    padding-top: 100px;
+    left: 0;
+    top: 0;
+    width: 100%;
+    height: 100%;
+    overflow: auto;
+    background-color: rgb(0, 0, 0);
+    background-color: rgba(0, 0, 0, 0.4);
+  }
+
+  .modal-content {
+    background-color: #fff;
+    margin: auto;
+    padding: 20px;
+    border: 1px solid #888;
+    width: 60%;
+  }
+
+  .project-img {
+    width: 40%;
+    margin-right: 25px;
+  }
```

Comment:
- [ ] It would be better to use the same demo image resolution that is provided in the Figma design to improve your pop-up design and to make sure that the UI design of your pop-up will not be broken after adding real images in the future.